website/content/docs/concepts/lease-explosions.mdx

```diff
@@ -0,0 +1,374 @@
+    --data @payload.json \
+    $VAULT_ADDR/v1/sys/quotas/lease-count/webapp-tokens
+   ```
+
+   This command is not expected to produce output, but you can read the user to confirm the settings.
+
+1. Confirm settings.
+
+   ```shell-session
+   $ curl \
+       --header "X-Vault-Token: $VAULT_TOKEN" \
+       --request GET \
+       --silent \
+       $VAULT_ADDR/v1/sys/quotas/lease-count/webapp-tokens \
+       | jq
+   ```
+
+   Example output:
+   
+   <CodeBlockConfig hideClipboard>
+   
+   ```json
+   {
+     "request_id": "188e22f1-dc1a-251a-a0a1-005e256fe70f",
+     "lease_id": "",
+     "renewable": false,
+     "lease_duration": 0,
+     "data": {
+       "counter": 0,
+       "inheritable": true,
+       "max_leases": 100,
+       "name": "webapp-tokens",
+       "path": "auth/approle/",
+       "role": "webapp",
+       "type": "lease-count"
+     },
+     "wrap_info": null,
+     "warnings": null,
+     "auth": null
+   }
+   ```
+
+   </CodeBlockConfig>
+
+</Tab>
+
+<Tab heading="CLI" group="cli">
+
+Write the webapp-tokens lease count quota.
+
+```shell-session
+$ vault write sys/quotas/lease-count/webapp-tokens \
+   max_leases=100 \
+   path="auth/approle" \
+   role="webapp"
+```
+
+Example output:
+
+<CodeBlockConfig hideClipboard>
+
+```plaintext
+Success! Data written to: sys/quotas/lease-count/webapp-tokens
+```
+
+</CodeBlockConfig>
+
+Confirm the setting.
+
+```shell-session
+$ vault read sys/quotas/lease-count/webapp-tokens
+```
+
+Example output:
+
+<CodeBlockConfig hideClipboard>
+
+```plaintext
+Key            Value
+---            -----
+counter        0
+inheritable    true
+max_leases     100
+name           webapp-tokens
+path           auth/approle/
+role           webapp
+type           lease-count
+```
+
+</CodeBlockConfig>
+
+</Tab>
+
+</Tabs>
+
+The limit is set to 100 leases for the AppRole auth method role named webapp.
+
+<Note>
+
+Enabling the rate limit audit logging may have an impact on the Vault performance if the volume of rejected requests is large.
+
+</Note>
+
+Review these resources for a deeper dive into controlling Vault resources:
+
+- [Vault resource quotas](/vault/docs/concepts/resource-quotas)
+
+- [Vault Enterprise lease count quotas](/vault/docs/enterprise/lease-count-quotas)
+
+- [Query audit device logs](/vault/tutorials/monitoring/query-audit-device-logs)
+
+#### Token type
+
+In some use cases, batch tokens can be a better fit for a use case than service tokens with respect to lease explosion. Review the following resources for help deciding when to use batch tokens and when to use service tokens:
+
+- [Vault service tokens vs batch tokens](/vault/tutorials/tokens/batch-tokens#service-tokens-vs-batch-tokens)
+
+- [Service vs batch token lease handling](/vault/docs/concepts/tokens#service-vs-batch-token-lease-handling)
+
+## Mitigating lease explosions
+
+Ultimately, the number of leases a system can handle is unique to the Vault deployment and environment. 
```

Comment:
Should we give a tiny bit more intro here. Maybe something like "The mitigation strategy you use will depend on what caused the explosion."? It also might be a good idea to be explicit about where users should start if they're trying to mitigate. For example, should they try these strategies in order or select the one that matches their problem? If they are supposed to choose one it might be nice to give them some symptoms to look for in order to choose, or a way to check if a strategy listed here is right for them.